appveyor.yml

```diff
@@ -9,7 +9,7 @@ install:
         $moduleName = 'ComputerManagementDsc'
         $mainModuleFolder = "Modules\$moduleName"
         Import-Module "$env:APPVEYOR_BUILD_FOLDER\DscResource.Tests\AppVeyor.psm1"
-        Invoke-AppveyorInstallTask -PesterMaximumVersion 4.0.8
+        Invoke-AppveyorInstallTask
```

Comment:
Why was pester restricted to version 4.0.8? Had do remove that because my tests were failing because of the too old Pester version. Is that a problem?

Reply:
All good - we were planning on removing this. There was a bug in a newer version of Pester that broke tests. The Pester team fixed this but we hadn't got round to backing the change out.